src/nvim/globals.h

```diff
@@ -1086,8 +1000,8 @@ EXTERN char psepcN INIT(= '/');            // abnormal path separator character
 EXTERN char pseps[2] INIT(= { '\\', 0 });  // normal path separator string
 #endif
 
-/* Set to TRUE when an operator is being executed with virtual editing, MAYBE
- * when no operator is being executed, FALSE otherwise. */
+// Set to bool when an operator is being executed with virtual editing, MAYBE
```

Comment:
I missed the `TriState` on this one, will revert to int for the time being. I intend to change the multiple tristates in the file as well in another PR.